tests/test_discretization.py

```diff
@@ -0,0 +1,67 @@
+import ibis
+import ibis.selectors as s
+import pytest
+
+import ibisml as ml
+
+
+@pytest.fixture()
+def train_table():
+    N = 100
+    return ibis.memtable(
+        {
+            "variable_col": list(range(N)),
+            "constant_col": [1.0] * N,
+            "str_col": ["value"] * N,
+        }
+    )
+
+
+@pytest.fixture()
+def test_table():
+    return ibis.memtable(
+        {
+            "variable_col": [float("-inf"), 0, 1, 12, float("inf")],
+            "constant_col": [float("-inf"), 1.0, 1.0, 1.0, float("inf")],
+            "str_col": ["value"] * 5,
+        }
+    )
+
+
+@pytest.mark.parametrize("k", [2, 5, 10, 100, 1000])
+@pytest.mark.parametrize("strategy", ["uniform", "quantile"])
+def test_KBinsDiscretizer(train_table, test_table, k, strategy):
```

Comment:
```suggestion
def test_discretize(train_table, test_table, k, strategy):
```
or something like that, rather than randomly adding camel case here